cetlvast/include/cetlvast/helpers.hpp

```diff
@@ -12,8 +12,48 @@
 
 #include <type_traits>
 #include <utility>
+#include <memory>
 
 #include "gtest/gtest.h"
 #include "gmock/gmock.h"
 
+#if defined(__clang__)
+#    if __has_feature(cxx_rtti)
+#        define CETLVAST_RTTI_ENABLED
+#    endif
+#elif defined(__GNUC__)
+#    if defined(__GXX_RTTI)
+#        define CETLVAST_RTTI_ENABLED
+#    endif
+#elif defined(_MSC_VER)
+#    if defined(_CPPRTTI)
+#        define CETLVAST_RTTI_ENABLED
+#    endif
+#endif
+
+namespace cetlvast
+{
+
+template <typename T>
+constexpr bool is_power_of_two(const T& value)
+{
+    return (value && !(value & (value - 1)));
```

Comment:
Do I understand correctly that the implicit cast to bool is allowed here only because it is not production code but test code, and the coding standard enforcement is relaxed here?